Presentation.Web/Controllers/API/DataProcessingRegistrationController.cs

```diff
@@ -577,6 +577,54 @@ public HttpResponseMessage PatchDataResponsibleRemark(int id, [FromBody] SingleV
                 .Match(_ => Ok(), FromOperationError);
         }
 
+        [HttpPatch]
+        [Route("{id}/oversight-completed")]
+        [SwaggerResponse(HttpStatusCode.OK)]
+        [SwaggerResponse(HttpStatusCode.Forbidden)]
+        [SwaggerResponse(HttpStatusCode.BadRequest)]
+        [SwaggerResponse(HttpStatusCode.NotFound)]
+        public HttpResponseMessage IsPatchOversightCompleted(int id, [FromBody] SingleValueDTO<YesNoUndecidedOption> completed)
```

Comment:
Hvad er det for et navn? Det er enten patch eller update den skal starte med. Ikke Is :)